reconcile/slack_cluster_usergroups.py

```diff
@@ -66,10 +66,12 @@ def get_desired_state(slack: SlackApi) -> dict[str, Any]:
         sendgrid=False, saas_files=False, aws=False, permissions=False
     )
     all_clusters = queries.get_clusters(minimal=True)
+    # this needs to be refactored - https://issues.redhat.com/browse/APPSRE-6575
     clusters = [
         c
         for c in all_clusters
-        if c.get("auth") and c["auth"].get("team") and c.get("ocm")
+        for auth in c["auth"]
+        if auth.get("team") and c.get("ocm")
```

Comment:
don't we get multiple times the same clusters here? Is that fine ?

Reply:
Yes, but this integration needs a bigger [refactoring](https://issues.redhat.com/browse/APPSRE-6575), because without GitHub as IDP we lose the `team` attribute. I'll address this in a later PR. With the current app-interface settings everything will be the same.

Reply:
ok, so the `auth.get("team")` actually catters for that it seems. LGTM